homeassistant/components/unifiprotect/icons.json

```diff
@@ -11,6 +11,9 @@
     },
     "remove_privacy_zone": {
       "service": "mdi:eye-minus"
+    },
+    "get_doorbell_user": {
```

Comment:
This isn't limited to doorbells AFAICT. Its all users?

Maybe `get_users`?

Reply:
Currently this only affects doorbells, but generally I agree with you, I am also generally not happy with the whole naming

Reply:
i renamed everything to get_user_keyring_info